packages/data-transport-layer/src/services/l1-ingestion/service.ts

```diff
@@ -230,14 +230,36 @@ export class L1IngestionService extends BaseService<L1IngestionServiceOptions> {
           targetL1Block,
         })
 
+        // We always try loading the deposit shutoff block in every loop! Less efficient but will
+        // guarantee that we don't miss the shutoff block being set and that we can automatically
+        // recover if the shutoff block is set and then unset.
```

Comment:
Don't you think its pretty ugly to nest an await like that and then call `toNumber` on its own line?

Reply:
Meh I don't think it's a huge problem. We're gonna toss all this code soon anyway.